web/controllers/task_controller.ex

```diff
@@ -40,12 +41,22 @@ defmodule CodeCorps.TaskController do
   def record(_conn, id), do: Task |> Repo.get(id)
 
   @spec handle_create(Plug.Conn.t, map) :: Ecto.Changeset.t
-  def handle_create(_conn, attributes) do
+  def handle_create(conn, attributes) do
+    project = Project |> Repo.get(attributes["project_id"])
+    if project.github_id do
+      current_user = Guardian.Plug.current_resource(conn)
+      github_id = github_module().create_issue(attributes, project, current_user)
+      attributes = Map.merge(attributes, %{"github_id" => github_id})
+    end
```

Comment:
I apologise if I'm reviewing too early and it was already your intention to do this, but this behavior would be what we'd expect to put into the task_service module described in #804